core/templates/dev/head/pages/exploration_player/audio_bar_directive.html

```diff
@@ -29,9 +29,29 @@
     background-color: #009688;
   }
 
-  .audio-toggle-button {
+  .fa-sort-up, .fa-sort-down {
```

Comment:
Prefix both of these with audio-bar? Otherwise this will apply to every usage of .fa-sort-* within the app.

Reply:
done